README.md

```diff
@@ -5,6 +5,26 @@
 This project will improve the FPGA code in the [M2 Cell Controller LabVIEW project](https://github.com/lsst-ts/ts_mtm2_cell).
 In that version, the VI named **4-port serial Master Slave.vi**, under `Chassis (cRIO-9038) >> FPGA Target (RIO0, cRIO-9038)` in the Project Explorer, interacts with the hardware inputs and outputs.
 
+## Platform
+
+- Windows 10
+- LabVIEW 2018 SP1 32-bit
+- cRIO-9049 (M2 simulator: m2-crio-simulator.ls.lsst.org)
```

Comment:
Please remove this IP address of `m2-crio-simulator.ls.lsst.org`.
We will remove this cRIO from base data center and put to the real M2 hardware soon.
The named IP address will be changed and can only be accessed by VPN at that time.

Thanks!